packages.json

```diff
@@ -21862,20 +21862,36 @@
     "description": "MTProto client written in Nim",
     "license": "MIT",
     "web": "https://github.com/nimgram/nimgram"
-  } , 
-   {
+  },
+  {
     "name": "json2xml",
     "url": "https://github.com/MhedhebiIssam/json2xml",
     "method": "git",
     "tags": [
       "json2xml",
       "json",
       "xml",
-      "XmlNode" , 
+      "XmlNode",
       "JsonNode"
     ],
     "description": "Convert json to xml  : JsonNode( comapatible with  module json ) To  XmlNode (comapatible with  module  xmltree)",
     "license": "MIT",
     "web": "https://github.com/MhedhebiIssam/json2xml"
+  },
+  {
+    "name": "nesper",
+    "url": "https://github.com/elcritch/nesper",
+    "method": "git",
+    "tags": [
+      "esp32",
+      "esp-idf",
+      "mcu",
+      "microcontroller",
+      "embedded",
+      ""
```

Comment:
Why an empty tag?

Reply:
Odd, I must’ve left a space at the end of the nimble tags prompt. Looks like it got turned into an empty tag.

Reply:
Ok, I removed it and solved merge conflicts. It should be ok now.